From 794713e88ee188d516e4956832a23f985a73917a Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Fri, 12 Nov 2021 10:07:44 -0500
Subject: Trac #29631: new spkg-configure.m4 for linbox.

This is essentially the file posted as an attachment on Trac 29631 by
Thierry Thomas. I've chosen to leave the lower bound at the latest
v1.6.3, since in the intervening months most distributions appear to
have updated.
---
 build/pkgs/linbox/spkg-configure.m4 | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 build/pkgs/linbox/spkg-configure.m4

diff --git a/build/pkgs/linbox/spkg-configure.m4 b/build/pkgs/linbox/spkg-configure.m4
new file mode 100644
index 00000000..fbf405c
--- /dev/null
+++ b/build/pkgs/linbox/spkg-configure.m4
@@ -0,0 +1,8 @@
+SAGE_SPKG_CONFIGURE([linbox], [
+  SAGE_SPKG_DEPCHECK([fflas-ffpack flint fplll givaro gmp iml m4ri m4rie mpfr ntl], [
+    PKG_CHECK_MODULES([LINBOX],
+                      [linbox >= 1.6.3],
+                      [],
+                      [sage_spkg_install_linbox=yes])
+  ])
+])
-- 
cgit v1.0-1-gd88e

